### ISSUE TITLE
Something is wrong in my Profile Readme Stats

### ISSUE THREAD

**Issue by CarlosViniMSouza** [anuraghazra/github-readme-stats#1148]
**Describe the bug**
Boy I don't know what happened, my Readme Stats was totally fine until last night. This morning after I performed my daily commits. I came across a Frame quite different from the original. (Python was 45% of the language bar, now it jumped to 98% instantly.)

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
![image](https://user-images.githubusercontent.com/60679101/122597390-aa16ba80-d041-11eb-9088-25df8f675514.png)

**Additional context**
In summary, the 'Most Used Languages' Board is weird!

**Comment by CarlosViniMSouza** [comment]
I think I've found the problem, and I'll take my question in the next update tonight.
